example/spec/loadTest/ingestLoad_queueGranPassThruSF.js

```diff
@@ -0,0 +1,253 @@
+const fs = require('fs');
+const pMap = require('p-map');
+const {
+  addCollections,
+  addProviders,
+  cleanupCollections,
+  cleanupProviders,
+} = require('@cumulus/integration-tests');
+const { updateCollection } = require('@cumulus/integration-tests/api/api');
+const { deleteExecution } = require('@cumulus/api-client/executions');
+
+const { LambdaStep } = require('@cumulus/integration-tests/sfnStep');
+const { getGranule } = require('@cumulus/api-client/granules');
+
+const { randomString } = require('@cumulus/common/test-utils');
+const {
+  waitForApiStatus,
+} = require('../helpers/apiUtils');
+
+const { buildAndExecuteWorkflow } = require('../helpers/workflowUtils');
+const {
+  loadConfig,
+  createTimestampedTestId,
+  createTestSuffix,
+  greenConsoleLog,
+  redConsoleLog,
+  yellowConsoleLog,
+  blueConsoleLog,
+} = require('../helpers/testUtils');
+
+const {
+  cleanupLoadTestGranules,
+  setupGranulesForIngestLoadTest,
+} = require('../helpers/granuleUtils');
+
+const workflowName = 'QueueGranulesPassthrough';
+const inputPayloadFilename = './spec/loadTest/ingestLoadTestPassthrough.input.payload.json';
+const inputPayloadJson = fs.readFileSync(inputPayloadFilename, 'utf8');
+
+const providersDir = './data/providers/s3/';
+const collectionsDir = './data/collections/s3_MOD09GQ_006';
+const granuleRegex = '^MOD09GQ\\.A[\\d]{7}\\.[\\w]{6}\\.006\\.[\\d]{13}$';
+
+// ** Configurable Variables
+
+const granuleCountPerWorkflow = 50; // 450 granules per workflow is the max allowable by the API
+const totalWorkflowCount = 8; // number of workflows to fire off
+
+const granuleCountThreshold = 0.95;
```

Comment:
What does this do?

Reply:
Added the comment for this variable.
// Percent granules to count as a successful batch on ingest timeout